### PR TITLE
session.created property

### PR DESCRIPTION
**Pull request by GomZik**
Issue #4: Add created property with timestamp indicating the time when session was created

### CODE REVIEW

**Comment by asvetlov** [comment]
At first I very appreciate your contribution.
But I think we should use a bit another data structure.
Instead of pushing `'created'` key into data dictionary we should create dictionary like:
```{'created': created_timestamp,
     'session': session_dict}
```
where current `data` moved into `session_dict`.
The benefits are:
1. we don't overlap session keys with our auxiliary ones. Say, user may want to have his own  `session['created']` value, why not?
2. I suspect `created` is not the only auxiliary session parameter. Say, we need to control session lifetime based on individual session info, not on system-wide value.

**Comment by GomZik** [comment]
Thanks for your review. I will rewrite that feature.

**Comment by GomZik** [comment]
Can you check my PR again, please?

File: aiohttp_session/__init__.py [hunk 1]
```diff
@@ -20,8 +20,16 @@ def __init__(self, identity, *, data=None, new=False):
...
+        if new or created is None:
+            self._created = int(time.time())
+        else:
+            self._created = created
+
+        if session_data is not None:
+            self._mapping.update(session_data)
 
     def __repr__(self):
         return '<{} [new:{}, changed:{}] {!r}>'.format(
```
Comment (asvetlov) [inline aiohttp_session/__init__.py]: Please add `created` info to the repr.

File: aiohttp_session/__init__.py [hunk 2]
```diff
@@ -36,6 +44,14 @@ def new(self):
     def identity(self):
         return self._identity
 
+    @property
+    def created(self):
+        return self._created
+
+    @property
+    def is_empty(self):
```
Comment (asvetlov) [inline aiohttp_session/__init__.py]: Please describe why do you need `is_empty` property.
What's your use case?

Comment (GomZik) [inline aiohttp_session/__init__.py]: Just a public property to determine is the session dict empty

File: aiohttp_session/__init__.py (hunk 1 above)
Comment (GomZik) [inline aiohttp_session/__init__.py]: Ok, i'l do it

File: aiohttp_session/__init__.py (hunk 2 above)
Comment (asvetlov) [inline aiohttp_session/__init__.py]: Ok than.
But name should be changed to `.empty` -- like `.changed` and `.created`

File: aiohttp_session/__init__.py [hunk 3]
```diff
@@ -143,6 +159,16 @@ def max_age(self):
     def cookie_params(self):
         return self._cookie_params
 
+    def get_session_data(self, session):
```
Comment (asvetlov) [inline aiohttp_session/__init__.py]: I believe it's private method, it should be named `_get_session_data` (starting with underscore).

**Comment by GomZik** [comment]
I made fixes from your comments

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**